python/doc/examples/functional_modeling/vectorial_functions/plot_createMultivariateFunction.py

```diff
@@ -0,0 +1,167 @@
+"""
+=============================
+Create multivariate functions
+=============================
+"""
+# %%
+#
+# Import the required modules
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build some multivariate functions :math:`f : \mathbb{R}^d \mapsto \mathbb{R}^q`.
+#
+# We can do that:
+#
+#     - Case 1: using a finite collection of multivariate functions :math:`f_i:  \mathbb{R}^d \mapsto \mathbb{R}^q`,
+#     - Case 2: using a tensorization of a univariate functions family,
+#     - Case 3: using a tensorization of an orthogonal univariate polynomials family,
+#     - Case 4: using a tensorization of an orthogonal univariate functions family.
+
+
+# %%
+# Case 1: we use a finite collection of multivariate functions
+# ============================================================
+#
+# In that case, we have a  finite collection of multivariate functions  :math:`f_i : \mathbb{R}^d \mapsto \mathbb{R}^q`.
+
+# We use :class:`~openturns.FiniteBasis` class.
+# We have to wait : issue #2201.
+#
+#
+# For example, we consider :
+#
+# .. math::
+#
+#     f_1(x_1, x_2)=(x_1^2, x_2^2)
+#     f_2: (x_1, x_2)= (x_1+1.0, 2.0*x_2)
+#     f_3: (x_1, x_2) = (cos(x_1x_2), x_2^3)
+
+# %%
+f1 = ot.SymbolicFunction(['x1', 'x2'], ['x1^2', 'x2^2'])
+f2 = ot.SymbolicFunction(['x1', 'x2'], ['x1+1.0', '2.0*x2'])
+f3 = ot.SymbolicFunction(['x1', 'x2'], ['cos(x1*x2)', 'x2^3'])
+
+# myBasis = ot.FiniteBasis([f1, f2, f3])
+# f = ot.FiniteBasis[1]
+
```

Comment:
All the functions in the basis must have the same input *and* output dimensions, isn'it? In this case, the input dimension of the three functions is equal to 2 and the same is true for the output dimension.

Reply:
yes: all the functions must have the same input dimension and the same output dimension.
In the current code, there is a bug because the construction of a Basis with functiuns with different input (output ) dimensions is allowed. I wrote an issue (#2274) on that point.

Reply:
In that case, I suggest to write this interesting analysis in the comments of the help page.